lib/filterable_by.rb

```diff
@@ -13,6 +13,28 @@ def self.normalize(value)
       end
     end
 
+    def self.merge(scope, unscoped, hash, name, **opts, &block)
+      key = name
+      positive = normalize(hash[key]) if hash.key?(key)
+      if positive.present?
```

Comment:
@andyborn here we only apply block/limit scope if `params['post_author_id']` is present